### ISSUE TITLE
Allow to specify the name of Primary Key

### ISSUE THREAD

**Issue by Takeno** [city41/node-sql-fixtures#44]
I've a database schema (MySQL in my case) which has no key named `id` but it uses `idcustomer` as primary key. In this way, `node-sql-fixtures` can't work. 

```
 ER_BAD_FIELD_ERROR: Unknown column 'id' in 'where clause'
```

It should be nice to have the option to specify the primary key of table to avoid error.

**Comment by city41** [comment]
I agree sql-fixtures should support this. But to be honest I'm not sure when I'd ever get time to implement it. I do welcome pull requests.
